spec/admin/target_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin Targets', type: :request do
+  let(:admin_user) { FactoryBot.create(:admin_user) }
+
+  before do
+    sign_in admin_user
+  end
+
+  describe 'Show all targets' do
+    it 'displays all targets for the admin user' do
+      target1 = FactoryBot.create(:target, title: 'Target 1')
+      target2 = FactoryBot.create(:target, title: 'Target 2')
+
+      get '/admin/targets'
+
+      expect(response).to have_http_status(200)
```

Comment:
As was mentioned in the PR for ticket 11, we can also use `be_successful` here

Reply:
Done!